Facebook.iOS/externals/Makefile

```diff
@@ -1,35 +1,34 @@
+POD=pod
+XCODEBUILD=/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild
 
-# We need to build the SDK as the prebuilt versions are 
-# using CLANG_MODULES_AUTOLINK.  This will cause linker
-# errors. We could use native references in the binding
-# but then we lose iOS 7 support.  Once the following is
-# fixed we can use the prebuilt SDK again.
-# https://bugzilla.xamarin.com/show_bug.cgi?id=44635
+PROJECT_ROOT=./Pods
+PROJECT=$(PROJECT_ROOT)/Pods.xcodeproj
+TARGETS=Bolts FBSDKCoreKit FBSDKShareKit FBSDKLoginKit FBSDKMessengerShareKit
 
-GIT_REPO_TAG=sdk-version-4.18.0
+all: Podfile.lock FacebookSDKStrings.bundle $(TARGETS)
 
-all: FBSDKCoreKit.a
+Podfile.lock:
+	$(POD) install
```

Comment:
I think that you forgot to add the Podfile to the commit.  Can you please check and add it?